citi-onboarding/maracatronics
# hotfix(robots-box-size): fix the class

## What i did
- Fixed the box robots size

## How to test
1. First, you need to use the command ```git fetch --all```
2. Change your branch to "hotfix/robots-box-size" with ```git checkout hotfix/robots-box-size```
3. Now, you need to open two terminals
4. In some terminal, go to client folder, with ```cd client```
5. In the other terminal, go to server folder with ```cd server```
6. In each terminal, run it with ```npm/yarn start``` in the client folder and ```nodemon index.js``` in the server.
7. Now, you need to open the http://localhost:3000/ (the client side)
8. Lastly, you check if the box size is right.
